metrics/metrics.go

```diff
@@ -60,6 +67,7 @@ func Middleware(registry prometheus.Registerer) gin.HandlerFunc {
 			"path":   c.FullPath(),
 			"status": strconv.Itoa(c.Writer.Status()),
 		}).Observe(time.Since(t).Seconds())
+		requestCount.Dec()
```

Comment:
might want a defer to make sure this is called on panics

Reply:
Good call! I made that change